src/CabanaPD_DisplacementProfile.hpp

```diff
@@ -0,0 +1,108 @@
+/****************************************************************************
+ * Copyright (c) 2022-2023 by Oak Ridge National Laboratory                 *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of CabanaPD. CabanaPD is distributed under a           *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef DISPLACEMENTPROFILE_H
+#define DISPLACEMENTPROFILE_H
+
+#include <Kokkos_Core.hpp>
+
+#include <Cabana_Core.hpp>
+
+namespace CabanaPD
+{
+
+auto getDim( const int dim )
+{
+    Kokkos::Array<int, 2> orthogonal;
+    orthogonal[0] = ( dim + 1 ) % 3;
```

Comment:
@streeve What does this do exactly?

Reply:
Given a dimension, returns the other two